spec/controllers/events_controller_spec.rb

```diff
@@ -25,15 +25,36 @@
 
 RSpec.describe EventsController, type: :controller do
 
+  before(:each) do
+    @user = FactoryBot.create :user
+    sign_in @user
+  end
+
   # This should return the minimal set of attributes required to create a valid
   # Event. As you add validations to Event, be sure to
   # adjust the attributes here as well.
   let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
+    {
+        name: "TT Event",
```

Comment:
Since I've seen this multiple times by now: please use `FactoryBot.build(:event).attributes` to create these hashes. Or if you want to make them specific for leagues or tournaments use `FactoryBot.build(:league/:event)` respectively. Makes it way easier to change the values in a single place if you do end up changing the model.